test_suite/README.md

```diff
@@ -25,172 +26,157 @@ These tests are used to validate ports of the API.
+1. Flash the application to the boards.
+1. Run the command `pytest`.<!--where? In the same location as the mbed compile command?-->
 
-Multiple tests cases, test files or test path can be specified. They are separated 
-by spaces.
+    The terminal displays the executed commands.
+1. When all tests have run, the console displays a summary of the results.
 
-```
-python -m pytest <test folder1> <test folder 2> <file path> <file path>::<test name>
-```
+## Run specific tests
+
+pytest allows you to run tests in a specific folder or file or individual tests:
 
-## List existing tests 
+* Folder: `pytest <folder name>`
+* Test file: `pytest <file path>`
+* Test within a file: `pytest <file path>::<test name>`
 
-To list tests in the test suite use the command: 
+To use multiple tests cases, test files or test paths, separate them with spaces:
 
 ```
-python -m pytest --collect-only
+pytest <test folder1> <test folder 2> <file path> <file path>::<test name>
 ```
 
-## Run test with markers
+## Run tests by markers
+<!--"run with" can imply the marker changes the test; "run by" implies selecting tests that have the marker, which I think is the case here-->
 
-Some of the tests has been annotated with a pytest marker. Four markers are 
-available: 
-- ble41: Test of Bluetooth 4.1 features
-- ble42: Test of Bluetooth 4.2 features
-- ble50: Test of Bluetooth 5.0 features
-- smoketest: Small set of smoke tests.
+Some of the tests has been annotated with a pytest marker:
 
-Run the command `python -m pytest -m <marker name>` to run the test anotated with `marker name`.
+- `ble41`: Test of Bluetooth 4.1 features.
+- `ble42`: Test of Bluetooth 4.2 features.
+- `ble50`: Test of Bluetooth 5.0 features.
+- `smoketest`: Small set of smoke tests.
 
-## Select targets type
+To run the test annotated with `marker name`, run the command `pytest -m <marker name>`.
 
-To select specific target types that should run the test, use the command line 
-argument `--platforms=`. Multiple platforms can be specified, they must be separated 
-with a `,`. 
+## Select targets
 
-For example, this command runs tests on `NRF52840_DK`: 
+To select specific targets for the test, use the command line
+argument `--platforms=`. To specify multiple platforms, separate them with a comma `,`.<!--this isn't a type - it's a specific board, isn't it?-->
+
+For example, to run tests on `NRF52840_DK`:
 
 ```sh
-python -m pytest --platforms=NRF52840_DK
+pytest --platforms=NRF52840_DK
 ```
 
-
 ## Flash targets
 
-It is possible to flash targets with a binary before the test start. Use the 
-argument `--flash=`. A list of platform/binary pair can be defined using the syntax
-`platform:binary`. Multiple pairs can be specified, they must be separated 
-with a `,`. 
+To flash targets with a binary before the test starts, use `--flash=`.<!--do I have to? is that part of the steps you described above for test running? or is it if I have something else I need on the board, and I can skip this test?-->
```

Comment:
It is an optional parameter to pass to pytest. It is not mandatory as users will probably flash when the board is compiled with mbed cli.

Reply:
```suggestion
If you did not use Mbed CLI to flash the binary to the targets before the test starts, use pytest to do it by passing the parameter `--flash=`.
```